Game.java

```diff
@@ -0,0 +1,16 @@
+
+public class Game extends State implements Cloneable {
```

Comment:
Could add javadoc here? Just so we don't forget why we extend this class int he first place.